### PR TITLE
Improve premeeting screens ux

### PR DESCRIPTION
**Pull request by quitrk**
Screenshot 2021-08-16 at 12 02 38
Screenshot 2021-08-16 at 12 03 03
e
Screenshot 2021-08-16 at 12 03 16

### CODE REVIEW

**Comment by quitrk** [comment]
https://github.com/jitsi/jitsi-meet-torture

File: css/premeeting/_premeeting-screens.scss [hunk 1]
```diff
@@ -96,130 +76,145 @@
...
+        text {
+            fill: white;
+            font-size: 26px;
+            font-weight: 400;
+        }
     }
 
     video {
-        height: 100%;
+        height: 100%;
```
Comment (saghul) [inline css/premeeting/_premeeting-screens.scss:216]: nit: trailing space

File: react/features/prejoin/components/Prejoin.js [hunk 2]
```diff
@@ -364,89 +320,79 @@ class Prejoin extends Component<Props, State> {
...
+                videoTrack = { videoTrack }>
+                <div
+                    className = 'prejoin-input-area'
+                    data-testid = 'prejoin.screen'>
+                    <InputField
+                        autoComplete = { 'name' }
+                        autoFocus = { true }
+                        className = { showError ? 'error' : '' }
+                        hasError = { showError }
+                        id = { 'Prejoin-input-field-id' }
```
Comment (saghul) [inline react/features/prejoin/components/Prejoin.js:336]: Do we need this id now that there is no label?

File: react/features/virtual-background/components/VideoBackgroundButton.js [hunk 3]
```diff
@@ -72,7 +73,8 @@ class VideoBackgroundButton extends AbstractButton<Props, *> {
 function _mapStateToProps(state): Object {
 
     return {
-        _isBackgroundEnabled: Boolean(state['features/virtual-background'].backgroundEffectEnabled)
+        _isBackgroundEnabled: Boolean(state['features/virtual-background'].backgroundEffectEnabled),
+        visible: checkBlurSupport()
```
Comment (saghul) [inline react/features/virtual-background/components/VideoBackgroundButton.js:77]: Nice!

**Review by saghul — COMMENTED** [review]
Code wise things look very good! (I glanced over CSS but I trust you) -- Here are some observations during testing:

- The connectivity checker makes the UI "jump". I was ready to click join, but it showed up and then my mouse was no longer on top of the join button.
- (This was the case before but still) Do we not want to have a logo in the prejoin screen?
- When transitioning from the prejoin screen to the lobby the UI "flickers" a bit becausse it briefly shows the full conference view. Can we avoid that abrupt transition?
- The avatar shown in the lobby waiting screen does not match what the use sees in the prejoin screen (if they configured their email address with a gravatar)
- The surfaces and "enter password" button have very little contrast between them. If you check with https://webaim.org/resources/contrastchecker/ you'll see it gives it a "Fail" for accessibility.

Last, can you please rebase? Your branch was unfortunately cut with a broken master on mobile, so I cannot easily test if mobile apps are still good.

**Comment by saghul** [comment]
After fixing an import (it's fixed in master already) the mobile app launches, but the lobby screen is never presented. (Ignore the amplitude error, it's inconsequential and I'll be fixing it shortly)

**Comment by quitrk** [comment]
Jenkins please test this please

**Comment by saghul** [comment]
The close button on mobile is misplaced. LGTM otherwise! Great work!

IMG_3688

**Review by saghul — APPROVED** [review]

**Pull request merged**